app/models/record.rb

```diff
@@ -35,36 +37,37 @@ def convert_rss_to_hash
 
   private
 
-  def parse_single_news_from_xml(xml_item)
-    {
-      author: xml_item.xpath("creator").try(:text),
-      full_version: false,
-      news_type: "news",
-      publication_date: publication_date(xml_item),
-      published_at: publication_date(xml_item),
-      source_url: {
-        url: xml_item.at_xpath("link").try(:text),
-        description: "source url of original article"
-      },
-      data_provider: data_provider,
-      contentBlocks: [
-        {
-          title: xml_item.at_xpath("title").try(:text),
-          body: xml_item.at_xpath("description").try(:text)
-        }
-      ]
-    }
-  end
+    def parse_single_news_from_xml(xml_item)
+      {
+        author: xml_item.xpath("creator").try(:text),
+        title: xml_item.at_xpath("title").try(:text),
+        full_version: false,
+        news_type: "news",
+        publication_date: publication_date(xml_item),
+        published_at: publication_date(xml_item),
+        source_url: {
+          url: xml_item.at_xpath("link").try(:text),
+          description: "source url of original article"
+        },
+        data_provider: data_provider,
+        contentBlocks: [
```

Comment:
konsequenter wäre `content_blocks` statt `contentBlocks`

Reply:
ja, nur wird daraus ja ein JSON welches dann in unser GraphQL übersetzt wird, und dort heißt es contentBlocks

Reply:
ich wundere mich aber, dass dort noch `data_provider` übergeben wird. das wird eigentlich gar nicht mehr benötigt